tap-version-14-specification.md

```diff
@@ -0,0 +1,1161 @@
+---
```

Comment:
@isaacs I approved it earlier today, before reviewing #138. I'm approving #138, where a header is added before the Markdown front-matter content. Maybe we should add one here too, with a mention to Artistic License 2.0? WDYT?

Reply:
9b5d0871345c6d0cb4d9095524cc4c2739579396 anyway, easy enough to keep it consistent.